templates/hcloud-csi.yml

```diff
@@ -103,7 +103,7 @@ spec:
       serviceAccountName: hcloud-csi
       containers:
         - name: csi-attacher
-          image: quay.io/k8scsi/csi-attacher:v1.2.1
+          image: quay.io/k8scsi/csi-attacher:v2.2.0
```

Comment:
It seems this version of `csi-attacher` no longer have `- --leader-election-type=leases`. The pod is now crashing because of it. It needs to be removed.

Reply:
sorry, i check my changes with original file and add all changes. 
And i squash commits.